docker-compose/keycloak/themes/shanoir-theme/login/login-update-password.ftl

```diff
@@ -12,6 +12,7 @@
                 <div class="${properties.kcInputWrapperClass!}">
                     <input type="password" id="password-new" name="password-new" class="${properties.kcInputClass!}"
                            autofocus autocomplete="new-password"
+						   style="color: red;"
```

Comment:
Why is that for ? 
![image](https://user-images.githubusercontent.com/21199767/231159455-110ce895-ec8a-4c8b-90f9-d5e960cca21b.png)